consensus/types/src/config_and_preset.rs

```diff
@@ -43,7 +43,6 @@ pub struct ConfigAndPreset {
 }
 
 impl ConfigAndPreset {
-    // DEPRECATED: the `fork_name` argument is never used, we should remove it.
     pub fn from_chain_spec<E: EthSpec>(spec: &ChainSpec, fork_name: Option<ForkName>) -> Self {
```

Comment:
Should we remove `fork_name` here given the comment above?

Reply:
I don't see `fork_name` getting passed from anywhere though, is it used by something other than lighthouse or am i missing something?